api/src/main/java/org/jboss/forge/roaster/model/util/Types.java

```diff
@@ -18,35 +18,26 @@
 import org.jboss.forge.roaster.model.Type;
 
 /**
- * Types utilities
+ * Util classes for java types.
  *
  * @author <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
  * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
  */
 public class Types
 {
-   private static final String BOOLEAN = "boolean";
 
-   // [B=byte,
-   // [F=float,
-   // [C=char,
-   // [D=double,
-   // [I=int,
-   // [J=long,
-   // [S=short,
-   // [Z=boolean
+   private static final Pattern SIMPLE_NAME_PATTERN = Pattern.compile("(?i)(?![0-9])[a-z0-9$_]+");
+   private static final List<String> PRIMITIVE_TYPES = Arrays.asList("Boolean", "Byte", "Double", "Float", "Integer",
+            "Long", "Short", "String");
+   // [B=byte, [F=float, [Z=boolean, [C=char, [D=double, [I=int, [J=long, [S=short,
    private static final Pattern CLASS_ARRAY_PATTERN = Pattern.compile("\\[+(B|F|C|D|I|J|S|Z|L)([0-9a-zA-Z\\.\\$]*);?");
-
-   private static final Pattern SIMPLE_ARRAY_PATTERN = Pattern
-            .compile("^((.)+)(\\[\\])+$");
+   private static final Pattern JAVA_SEPERATOR_PATTERN = Pattern.compile("\\.");
```

Comment:
s/SEPERATOR/SEPARATOR

Reply:
What is this patern for? Package names? File Names?

Reply:
fixed + adding comment